packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -33,15 +33,20 @@ export interface TransactionFunctions {
 	): boolean;
 }
 
+export enum Status {
+	FAIL,
```

Comment:
first enum should be numerically initialized e.g. `export enum Status {
	OK = 1,
	FAIL,
}`